application/controllers/admin/SurveysGroupsController.php

```diff
@@ -123,25 +178,49 @@ public function update($id)
         $this->_renderWrappedTemplate('surveysgroups', 'update', $aData);
     }
 
+    /**
+     * Show the survey settings menue for a particular group
+     * @param integer $id group id, used for permission control
+     * @return void
+     */
+    public function surveysettingmenues($id) {
```

Comment:
Stick to camelCase would be best. So: surveySettingsMenus. Also, it's a good idea to always include a verb, like "showSurveySettingMenus", or "editSurveySettingMenus".

Reply:
I copy paste existing global function … then rename both ?

Reply:
Ah. OK, let me checkout this branch and check...

Reply:
Gah. OK, just add a todo for now. Also, missing verb in function name.

Reply:
I prefer to make it after merge (on both).

See https://github.com/LimeSurvey/LimeSurvey/pull/1632#discussion_r547188628 .

I already have some idea : 

1. Create more Permission view and Permission function : use it in SurveysGroups and Surveys
2. Fix https://bugs.limesurvey.org/view.php?id=14551 (in fact i start SurveysGroups with this system, but if Survey don't have it : SurevsysGroups must not have it)
3. etc …

Reply:
Alright.